l10n_it_fatturapa_out/tests/test_fatturapa_xml_validation.py

```diff
@@ -924,3 +924,49 @@ def test_max_invoice_in_xml(self):
         res = self.run_wizard(invoices.ids)
         attachments = self.attach_model.browse(res["res_id"])
         self.assertEqual(len(attachments), 1)
+
+    def test_preventive_checks(self):
+        invoice_form = Form(
+            self.env["account.move"].with_context({"default_move_type": "in_invoice"})
+        )
+        invoice = invoice_form.save()
+        with self.assertRaises(UserError) as ue:
+            self.run_wizard(invoice.id)
+        self.assertIn(invoice.name, ue.exception.args[0])
```

Comment:
Puoi chiarire quale dev'essere l'errore sollevato qui e negli `assert` successivi? 
Devono essere errori per i nuovi check sul campo `city` o `fiscal_document`, o forse si vuole verificare che salti uno dei check già esistenti?

Reply:
Adesso faccio il controllo anche sul testo del messaggio (che è autoesplicativo). Dovrebbe cambiare con la localizzazione ma i test in github credo possiamo assumere siano in inglese.

Reply:
Ho scritto il test per tutti i check, uno per uno. In precedenza non veniva fatto nessun test, anche se i check c'erano.